.github/workflows/build.yml

```diff
@@ -0,0 +1,44 @@
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build:
+    name: Push Docker Image to ECR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+          role-duration-seconds: 900
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          # list of Docker images to use as base name for tags
+          images: ${{ secrets.ECR_REPO }}
+          # generate Docker tags based on the following events/attributes
+          tags: |
+            type=ref,event=branch,prefix=branch-
+            type=ref,event=tag
+            type=sha
+      - name: Set up Docker Buildx
```

Comment:
```suggestion
      - name: Set up Docker Build
```

Reply:
Or is `Buildx` correct?, I see it in next line, so maybe not a typo.

Reply:
Buildx is correct. https://docs.docker.com/buildx/working-with-buildx/

Reply:
Ah! Thx.